microsoftgraph/microsoft-graph-explorer-v4
# Fix: Resource path causing app to crash

## Overview

fixes #3048 

### Demo

https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/18407044/ba3c366b-b5d3-437d-a7de-260b4508a5a4


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Check out branch
* Load GE
* type https://graph.microsoft.com/v1.0/oauth2PermissionGrants
* Observe page doesn't crash
